src/Components/Basic/Markdown/Styles.ts

```diff
@@ -4,12 +4,37 @@ import {
 } from 'react-native';
   
 export const Styles = StyleSheet.create({
+    h1: {
+        fontSize: 32,
+    } as TextStyle,
+    h2: {
+        fontSize: 24,
+    } as TextStyle,
+    h3: {
+        fontSize: 18,
+    } as TextStyle,
+    h4: {
+        fontSize: 16,
+    } as TextStyle,
+    h5: {
```

Comment:
Please check whether Markdown config was included in card config.

Reply:
there is no config about Markdown before.